app/javascript/article-form/components/EditorBody.jsx

```diff
@@ -70,6 +70,13 @@ export const EditorBody = ({
     }
   });
 
+  // In some platforms we've seen `navigator.clipboard` return `undefined`
+  // despite `window.isSecureContext` being true. A refresh fixes the problem.
+  // It's important to only do this when isSecureContext (not in dev/test env)
+  if (navigator.clipboard == null && window.isSecureContext) {
+    window.location.reload();
+  }
+
```

Comment:
This fixes the edge case we're encountering in the iOS app. The `reload()` is only triggered when there is this mismatch of  `navigator.clipboard` being undefined when `window.isSecureContext === true`.

I'm open to hearing thoughts/alternatives. Maybe we can "disable InstantClick" when visiting `/new`? I'm not sure if this is possible or if it will fix the issue, but I've found this fix to work.

Reply:
Won't this cause the user to lose their draft in progress if it's triggered?

Reply:
Marking the conversation as resolved since this change is no longer included 🙂